torch_xla/csrc/tensor.cpp

```diff
@@ -603,43 +603,139 @@ bool XLASymNodeImpl::is_float() {
   return false;
 }
 
-bool XLASymNodeImpl::bool_() {
-  auto dn = torch_xla::DimCast(node());
-  return dn->getDynamicValue() != 0;
+c10::SymNode XLASymNodeImpl::add(const c10::SymNode& other) {
+  auto p_other = dynamic_cast<XLASymNodeImpl*>(other.get());
+  auto n_add = torch::lazy::MakeNode<SizeAdd>(node(), p_other->node());
+  return c10::make_intrusive<XLASymNodeImpl>(n_add);
```

Comment:
can we add test for this method?

Reply:
Another [pr](https://github.com/pytorch/xla/pull/4258) is intended to add tests for this method and the current pr just wants to force fail the unimplemented method. Do you mind if I add the test in the other pr?

Reply:
oh ok, you did not modify this implemention, it is just a git diff thing.